src/app/store/store.ts

```diff
@@ -25,4 +25,11 @@ const store = createStore<RootState, any, {}, {}>(
   composeEnhancers(applyMiddleware(thunk))
 );
 
+// Allow hot-reloading reducers
```

Comment:
Oh yeah - the React Router docs had an example of how to enable hot reload for reducer code too, so here it is.